buildscripts/semantic-convention/templates/SemanticAttributes.java.j2

```diff
@@ -64,10 +64,10 @@ public final class {{class}} {
   {%- for attribute in attributes if attribute.is_local and not attribute.ref %}
 
   /**
-  * {% filter escape %}{{attribute.brief | to_doc_brief}}.{% endfilter %}
+  * {% filter safe %}{{attribute.brief | render_markdown}}{% endfilter %}
```

Comment:
```suggestion
  * {{attribute.brief | render_markdown | safe}}
```

Reply:
I don't even think it needs the `| safe` which I believe is a no-op.